cordova-lib/package.json

```diff
@@ -40,14 +38,19 @@
     "tar": "1.0.2",
     "underscore": "1.7.0",
     "unorm": "1.3.3",
-    "valid-identifier": "0.0.1",
-    "xcode": "0.8.0"
+    "valid-identifier": "0.0.1"
   },
+  "bundledDependencies": [
+    "cordova-common"
+  ],
   "devDependencies": {
     "istanbul": "^0.3.4",
     "jasmine-node": "1.14.5",
     "jshint": "2.5.8",
-    "rewire": "2.1.3"
+    "rewire": "2.1.3",
+    "osenv": "0.1.0",
```

Comment:
My bad. Returned plist and xcode back to 'dependencies' section and totally got rid of osenv module.